lib/fastlane/plugin/rome/actions/rome_action.rb

```diff
@@ -219,6 +231,15 @@ def self.available_options
                                        optional: true,
                                        verify_block: proc do |value|
                                          UI.user_error!("Please pass a valid value for noskipcurrent. Use one of the following: true, false") unless value.kind_of?(TrueClass) || value.kind_of?(FalseClass)
+                                       end),
+          
+          FastlaneCore::ConfigItem.new(key: :concurrently,
```

Comment:
`CONCURRENTLY` or `CONCURRENCY`

Reply:
`CONCURRENTLY` . Did I make a typo?

Reply:
`rome <upload/download> --concurrently` this is the command

Reply:
😀👍